src/components/title.js

```diff
@@ -2,8 +2,14 @@ const loader = new THREE.FontLoader();
 const fontJSON = require("../assets/Acme_Regular.json");
 const font = loader.parse(fontJSON);
 AFRAME.registerComponent("title", {
+  schema: {
+    value: { type: "string", default: "Hello World!" },
+    color: { type: "number", default: 0xf7cb0d },
+    metalness: { type: "number", default: 0.1 },
+    roughness: { type: "number", default: 0.2 },
+  },
   init: function () {
-    const geometry = new THREE.TextGeometry("Welcome to MathVR", {
+    const geometry = new THREE.TextGeometry(this.data.value, {
       font: font,
       size: 1,
```

Comment:
Would it be fair to allow the users to customize the font/size/height or any other variables here?

Reply:
Yeah I was thinking of that as well. Let me see if I can do that